.github/actions/javascript/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -67,6 +67,7 @@ function getDeployMessage(deployer, deployVerb, prTitle) {
  */
 function commentPR(PR, message) {
     return GithubUtils.createComment(context.repo.repo, PR, message)
+        .then(ActionUtils.sleep(1000))
```

Comment:
This is the main place we are commenting on many issues/ PRs from here so I think tis hould be fine added only here to start with as maybe the rate limiting wont be such problem in future anymore as we should not have such big deploys as we had when this issue occurred.

Working with JS Promises is not my strength so just wanna highlight this to make sure its correctly implemented

Reply:
I think you have to provide `() => ActionUtils.sleep(1000)`, I created a small mock of the code and it was not working and the mentioned change made it work - https://jsfiddle.net/abdulrahuman5196/Loupsxwc/ . Let me know if there is something wrong with the mock.

Reply:
Thanks! that makes sense, update the code 🙇